src/api/app/controllers/webui/user_controller.rb

```diff
@@ -81,19 +81,30 @@ def requests
         3 => :creator,
         5 => :priority
       }
-    sorting_field = sortable_fields[params[:iSortCol_0].to_i]
+    sorting_field = sortable_fields[params['order[0][column]'].to_i]
     sorting_field ||= :created_at
-    sorting_dir = params[:sSortDir_0].try(:to_sym)
-    sorting_dir = :asc unless ["asc", "desc"].include?(params[:sSortDir_0])
-    @requests = @displayed_user.requests(params[:search])
+    sorting_dir = params['order[0][dir]'].try(:to_sym)
+    sorting_dir = :asc unless ["asc", "desc"].include?(params['order[0][dir]'])
+    search = ""
+    search = params[:search].permit![:value] if params[:search]
```

Comment:
this two lines can be written like that:

`search = params[:search] ? params[:search].permit![:value] : ""`